ocaml/parsing/jane_syntax.ml

```diff
@@ -41,8 +41,28 @@ open Jane_syntax_parsing
 
 module With_attributes = With_attributes
 
+(* So far, all pieces of Jane Syntax use the same erasability/feature for
+   all the sub-ASTs involved in that syntax. This type captures this
+   per-Jane-Syntax configuration.
+*)
+type jane_syntax_config =
+  { erasability : Jane_syntax_parsing.Embedded_name.Erasability.t
+  (** The erasability component of the embedded name. *)
+  ; feature : Jane_syntax_parsing.Feature.t
+  (** The feature component of the embedded name. *)
+  }
+
+let make_for_language_extension language_extension =
+  { erasability =
+      if Language_extension.is_erasable language_extension
+      then Erasable
+      else Non_erasable
+  ; feature = Language_extension language_extension
+  }
+
 (** List and array comprehensions *)
 module Comprehensions = struct
+  let { erasability; feature } = make_for_language_extension Comprehensions
   let extension_string = Language_extension.to_string Comprehensions
```

Comment:
Why don't we put `extension_string` in `jane_syntax_config` as well?  Then this is just one line, and I think it's even clearer!

Reply:
I did this as you recommended, but then this point became moot due to another refactor that pushed me to remove `jane_syntax_config` altogether.